lib/interface/JsonObject.dart

```diff
@@ -59,4 +59,21 @@ abstract class JsonObject {
     });
     return result;
   }
+
+  /// Returns all values as a String separated by a hyphen
+  /// value1 - value2 - value3
+  String toValueString() {
```

Comment:
is this proxy method needed? The callers can just call JsonObject.toValueString(json) ?

Reply:
Thanks for your review @jasmeet0817, I just added it that way because its implemented the same for `toData` a few lines above, but I have no problem if we update it later on.